src/VisualStudio/Core/Test/Options/BasicEditorConfigGeneratorTests.vb

```diff
@@ -73,11 +74,18 @@ dotnet_style_prefer_is_null_check_over_reference_equality_method = true:suggesti
 # Field preferences
 dotnet_style_readonly_field = true:suggestion
 
+# Parameter preferences
+dotnet_style_unused_parameters = all:suggestion
+
 #### VB Coding Conventions ####
 
 # Modifier preferences
 visual_basic_preferred_modifier_order = partial,default,private,protected,public,friend,notoverridable,overridable,mustoverride,overloads,overrides,mustinherit,notinheritable,static,shared,shadows,readonly,writeonly,dim,const,withevents,widening,narrowing,custom,async,iterator
 
+# Expression-level preferences
+visual_basic_style_unused_value_assignment_preference = unused_local_variable:suggestion
+visual_basic_style_unused_value_expression_statement_preference = unused_local_variable:silent
```

Comment:
1. why is this visual_basic specific, and not a dotnet concern?

Reply:
Because the default values are different. Discard for C# (which is unsupported for VB) and Unused local variable for VB.

---
In reply to: [230606129](https://github.com/dotnet/roslyn/pull/30777#discussion_r230606129) [](ancestors = 230606129)

Reply:
Ah i see.  That's definitely an interesting case.  Where the languages can be different, but overlap.  This makes sense now.

Reply:
I started with single dotnet option in the core layer, but had to make it different options primarily due to this restriction.

---
In reply to: [230988344](https://github.com/dotnet/roslyn/pull/30777#discussion_r230988344) [](ancestors = 230988344)